src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -21,13 +21,23 @@ class Day_View extends View {
 
 	/**
 	 * Slug for this view
+	 * @todo deprecate and use static::$view_slug
```

Comment:
Here and elsewhere this property is deprecated, why not just use the `@deprecated` annotation? It will not throw in WordPress, but it will report to anyone using the property is deprecated in IDE context.

Reply:
We're still using it in some places - I was going to deprecate it iteratively.